generate-models-maven-plugin/src/main/java/com/onfido/models/Property.java

```diff
@@ -89,7 +89,7 @@ private static TypeInfo typeInfoFor(PropertyJson propertyJson) {
       case "integer":
         return TypeInfo.primitiveType("int", "Integer", "0");
       case "object":
-        return TypeInfo.objectType("Map<String, Object>");
```

Comment:
Can you share an example where this is causing parsing issues? Otherwise I think this will break the current behaviour